src/main/java/com/spotinst/sdkjava/model/api/azure/statefulNode/ApiLaunchSpecExtensionsProtectedSettings.java

```diff
@@ -0,0 +1,49 @@
+package com.spotinst.sdkjava.model.api.azure.statefulNode;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.Set;
+/**
+ * Created by Bansi Chapla on 23/11/2021.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiLaunchSpecExtensionsProtectedSettings implements IPartialUpdateEntity {
```

Comment:
Where did u get the fields for this object?

Reply:
used in extensions

Reply:
from postman collection shared by dev team